src/imageio/format/j2k.c

```diff
@@ -395,9 +395,15 @@ int write_image(dt_imageio_module_data_t *j2k_tmp, const char *filename, const v
 //        }
 //        break;
       case 12:
-        for(int i = 0; i < w * h; i++)
+#ifdef _OPENMP
+#pragma omp parallel for simd default(none) \
+  dt_omp_firstprivate(in, w, h, image, numcomps) \
+  schedule(simd:static) \
+  collapse(2)
+#endif
+        for(int i = 0; i < w * h; ++i)
         {
-          for(int k = 0; k < numcomps; k++)
+          for(int k = 0; k < numcomps; ++k)
```

Comment:
Is the `k++` to `++k` because of "something" or just style?

Reply:
[Force of habit I suppose](https://stackoverflow.com/questions/24901/is-there-a-performance-difference-between-i-and-i-in-c), though it probably doesn't matter here. Happy to revert if we want to minimize the diff...

Reply:
Let's keep this here, but really we don't need such change as it doesn't change anything except having lager diff :)